daml-lf/spec/daml-lf-2.rst

```diff
@@ -4202,21 +4161,21 @@ BigNumeric functions
   Adds the two decimals. Throws an ``ArithmeticError`` if the output
   is not a valid BigNumeric.
 
-  [*Available in version ≥ 1.13*]
+  [*Available in version ≥ 1.dev*]
```

Comment:
Shouldn't that be 2.1? (Here and below.)

Reply:
no 2.dev